### PR TITLE
[Opentext Documentum] Connector client documentation

### PR DESCRIPTION
**Pull request by moxarth-elastic**
## Part of https://github.com/elastic/connectors/issues/2163

Connector client documentation for Opentext Documentum connector.

#### Pre-Review Checklist
- [x] this PR does NOT contain credentials of any kind, such as API keys or username/passwords (double check `config.yml.example`)
- [x] this PR has a meaningful title
- [x] this PR links to all relevant github issues that it fixes or partially addresses
- [x] if there is no GH issue, please create it. Each PR should have a link to an issue
- [x] this PR has a thorough description
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] Considered corresponding documentation changes

### CODE REVIEW

File: docs/reference/opentext_documentum.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Opentext Documentum Connector
+
+The Elastic Opentext Documentum connector is written in Python using the [Elastic connector framework](https://github.com/elastic/connectors/tree/main). View the [source code for this connector](https://github.com/elastic/connectors/blob/main/connectors/sources/opentext_documentum.py).
+
+## Availability and prerequisites
+
+This connector is available as a self-managed connector client.
+To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is an **example connector** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties.
```
Comment (jedrazb) [inline docs/reference/opentext_documentum.md:10]: tech preview

File: docs/reference/opentext_documentum.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Opentext Documentum Connector
+
+The Elastic Opentext Documentum connector is written in Python using the [Elastic connector framework](https://github.com/elastic/connectors/tree/main). View the [source code for this connector](https://github.com/elastic/connectors/blob/main/connectors/sources/opentext_documentum.py).
+
+## Availability and prerequisites
+
+This connector is available as a self-managed connector client.
+To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is an **tech preview connector** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties.
```
Comment (jedrazb) [inline docs/reference/opentext_documentum.md:10]:
```suggestion
This connector is a **tech preview connector** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties.
```

**Review by jedrazb — APPROVED** [review]
one small change suggested but otherwise LGTM, let's wait for @leemthompo to take a look as well

File: docs/reference/opentext_documentum.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+#### `Opentext Documentum host url`  (required)
+
+The domain where Opentext Documentum is hosted. Example:
+
+- `https://192.158.1.38:2099/`
+
+#### `Username`  (required)
+
+The username of the account to connect with Opentext Documentum.
```
Comment (leemthompo) [inline docs/reference/opentext_documentum.md:38]:
```suggestion
The username of the account to connect to Opentext Documentum.
```

File: docs/reference/opentext_documentum.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+#### `Password`  (required)
+
+The password of the account to connect with Opentext Documentum.
```
Comment (leemthompo) [inline docs/reference/opentext_documentum.md:42]:
```suggestion
The password of the account to connect to Opentext Documentum.
```

File: docs/reference/opentext_documentum.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+#### `Repositories`
```
Comment (leemthompo) [inline docs/reference/opentext_documentum.md:44]: Optional or required?

File: docs/reference/opentext_documentum.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+Examples:
+
+- `elastic`, `kibana`
+- `*`
+
+#### `Enable SSL`
+
+Enable SSL for the Opentext Documentum instance.
+
+#### `SSL Certificate`
```
Comment (leemthompo) [inline docs/reference/opentext_documentum.md:59]: Depends on previous?

File: docs/reference/opentext_documentum.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+- Files bigger than 10 MB won't be extracted.
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to the destination Elasticsearch index.
+
+### Sync types
+[Full syncs](https://www.elastic.co/guide/en/enterprise-search/current/connectors-sync-types.html#connectors-sync-types-full) are supported by default for all connectors.
+
+This connector currently does not support [incremental syncs](https://www.elastic.co/guide/en/enterprise-search/current/connectors-sync-types.html#connectors-sync-types-incremental).
+
+## Sync rules
+
```
Comment (leemthompo) [inline docs/reference/opentext_documentum.md:92]: Advanced sync rules?

**Review by leemthompo — COMMENTED** [review]
Few nits but looking good! :)

File: docs/reference/opentext_documentum.md (hunk 7 above)
Comment (moxarth-elastic) [reply docs/reference/opentext_documentum.md:92]: This connector does not have advanced sync rules.

File: docs/reference/opentext_documentum.md (hunk 5 above)
Comment (moxarth-elastic) [reply docs/reference/opentext_documentum.md]: updated with optional 👍

File: docs/reference/opentext_documentum.md (hunk 7 above)
Comment (leemthompo) [reply docs/reference/opentext_documentum.md:92]: Cool, let's make that explicit in docs so :)

Comment (moxarth-elastic) [reply docs/reference/opentext_documentum.md:92]: Done, added.

**Review by leemthompo — APPROVED** [review]

**Comment by leemthompo** [comment]
@moxarth-elastic @jedrazb  should this be backported to 814?

**Comment by leemthompo** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.14|[](https://github.com/elastic/connectors/pull/2447)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**